templates/dart/lib/enums.dart.twig

```diff
@@ -7,11 +7,3 @@ extension HttpMethodString on HttpMethod {
     return this.toString().split('.').last.toUpperCase();
   }
 }
```

Comment:
Don't we need to completely remove this template file?

Reply:
Same, HttpMethod emum is used internally and required